src/test/java/it/gov/pagopa/payment/service/qrcode/QRCodeConfirmServiceTest.java

```diff
@@ -26,6 +27,7 @@ class QRCodeConfirmServiceTest {
   @Mock private TransactionInProgressRepository repositoryMock;
   @Mock private TransactionNotifierService notifierServiceMock;
   @Mock private ErrorNotifierService errorNotifierServiceMock;
+  @Mock private AuditUtilities auditUtilities;
```

Comment:
When it's a mock, let's explicitly it in the name
```suggestion
  @Mock private AuditUtilities auditUtilitiesMock;
```